Python/use_cases/rube_goldberg/rube_goldberg.py

```diff
@@ -63,12 +65,22 @@ def __init__(self):
         # Parse the default objects.csv spreadsheet. 
         self.object_audio_data = PyImpact.get_object_info()
 
+        self.temp_amp = 0.1
+
+        # Keep track of the current trial number, for logging purposes.
+        self.current_trial_num = 0
+
         # Fetch the ball and board model's records; we will need them later to change its material.
         self.special_models = ModelLibrarian(library="models_special.json")
         self.full_models = ModelLibrarian(library="models_full.json")
         self.ball_record = self.special_models.get_record("prim_sphere")
         self.board_record = self.full_models.get_record("wood_board")
 
+        # Set path to write out logging info.
+        self.root_dest_dir = Path("dist/mode_properties_logs")
+        if not self.root_dest_dir.exists():
+            self.root_dest_dir.mkdir(parents=True)
+

```

Comment:
You should remove this and disable logging so that this controller doesn't generated unwanted files.

Reply:
@alters-mit This is also now a demo of logging, so why remove these lines?  This is from your standard "boiler plate" logging process....